### ISSUE TITLE
[Feature Request] Support other options than left, right, center in table cell alignment

### ISSUE THREAD

**Issue by del2z** [AnubisNekhet/AnuPpuccin#90]
**Feature Description**
The custom table style only supports single alignment of cell content (left, center or right), ignoring the markers in markdown table.

```markdown
| head1 | head2 | head3|
|:------|:-------:|------:|
| a | b | c |
| d | e | f |
```

If `<td> Text Align` is set to center, all contents will be centered, which is not I want.
Hope to fix this.

**Comment by AnubisNekhet** [comment]
Fixed with the latest commit, markdown arguments will override style settings in <td> alignment.

**Comment by AnubisNekhet** [comment]
Implemented with v1.4.0!
